invoice/src/main/java/org/killbill/billing/invoice/dao/InvoiceDaoHelper.java

```diff
@@ -281,7 +285,7 @@ public List<InvoiceModelDao> getAllInvoicesByAccountFromTransaction(final Boolea
                                                        .filter(invoice -> includeVoidedInvoices || !InvoiceStatus.VOID.equals(invoice.getStatus()))
                                                        .collect(Collectors.toUnmodifiableList());
         if (includeInvoiceComponents) {
-            populateChildren(filtered, invoicesTags, false, entitySqlDaoWrapperFactory, context);
+            populateChildren(filtered, invoicesTags, false, true, entitySqlDaoWrapperFactory, context);
```

Comment:
Good. For pass #2, this probably one option where we could optimize based on use cases, internal vs externals.